.github/workflows/checkAndSubmitAddonMetadata.yml

```
@@ -278,16 +263,40 @@ jobs:
         jqExitCode=$?
         rm discussions.old.json
         exit $jqExitCode
+    - name: Create review comment
+      id: createComment
+      if: always()
+      uses: actions/github-script@v6
+      with:
+        script: |
+          const fs = require('fs')
+          const addonVersion = "${{ steps.getAddonNameAndVersion.outputs.addonVersion }}"
+          const contents = fs.readFileSync('discussions.json')
+          const data = JSON.parse(contents)
+          const addonName = $ {{ steps.getAddonNameAndVersion.outputs.addonName }}
+          const discussionId = data[addonName]["discussionId"]
+          const mutation = `mutation {
+            addDiscussionComment(input: {body: "Reviews for version ${addonVersion}", discussionId: "${discussionId}", clientMutationId: "GitHub Script action"}) {
```

lint for readability

```suggestion
            addDiscussionComment(
                        input: {
                                    body: "Reviews for version ${addonVersion}",
                                    discussionId: "${discussionId}",
                                    clientMutationId: "GitHub Script action"
                         }
            ) {
```